compiler/ballerina-parser/src/test/resources/expressions/access-expr/member_access_expr_source_11.bal

```diff
@@ -0,0 +1,5 @@
+public function main() {
+    int[] a = [];
+    a[2wed2] = 10;
+    a[2147483647-9] = 10;
```

Comment:
`a[2147483647-9] = 10;` is recovering as `MISSING[] INVALID[]] = 10;`
can we expect `a[2147483647  INVALID[-] INVALID[9] ] = 10;` as before?

Reply:
Previously it was recovered as a list binding pattern, hence cases the CCE in NodeTransformer. Will check whether we can improve this

Reply:
Parser will recover `a[2147483647-9] = 10;` as `a[2147483647][-9] = 10;` as the best match rather than removing. So, it will insert the first `[` then the recovery of `a[2147483647-9] = 10;` is same as `a[2147483647] - 9] = 10;` by seeing `-` token it will go on the binary expression path. So in the end, it will be an invalid expr in LHS.

Reply:
ACK.